awxkit/awxkit/cli/docs/source/examples.rst

```diff
@@ -57,5 +57,5 @@ Importing an SSH Key
 
     awx credentials create --credential_type 'Machine' \
         --name 'My SSH Key' --user 'alice' \
-        --inputs "{'username': 'server-login', 'ssh_key_data': '@~/.ssh/id_rsa`}"
+        --inputs "{'username': 'server-login', 'ssh_key_data': '""$(cat ~/.ssh/id_rsa)""'}"
```

Comment:
Does this not work?  This is an explicit feature of the AWX CLI that _should_ be working (if it's not, it represents a bug).

Reply:
beside of the fact, that the 3rd character from the end is a backtick (`) instead of a single coding sign (') - at this point I want to say thank you to the colleague helping me with these ***-characters....
I'm getting:
```
# awx credentials create --credential_type 'Machine'     --name 'My SSH Key' --user 'admin'     --inputs "{'username': 'server-login', 'ssh_key_data': '@~/.ssh/id_rsa'}" -k
{"inputs": {"ssh_key_data": ["Invalid certificate or key: @~/.ssh/id_rsa..."]}}
```

Reply:
Ah, I see.  Yea, that backpack *is* a typo.

Reply:
If you're seeing this error, this could be a bug in the CLI - I'll take a peek shortly.

Reply:
@mwiora

I think I see what's up.  These instructions have a typo, but *also* if the file path you specify _doesn't_ exist, it just ignores that and uses the raw string instead.  I'll upload a PR to address this.

Reply:
https://github.com/ansible/awx/pull/4881

Reply:
but this seems not to be the full magic... - without testing the variant with your modifications I've retestet, if I would be using a missing file
```
root@awxhost:/home/pix# awx credentials create --credential_type 'Machine'     --name 'My SSH Key' --user 'admin'     --inputs "{'username': 'server-login', 'ssh_key_data': '@/root/.ssh/id_rsa'}" -k
{"inputs": {"ssh_key_data": ["Invalid certificate or key: @/root/.ssh/id_rsa..."]}}
root@awxhost:/home/pix# awx credentials create --credential_type 'Machine'     --name 'My SSH Key' --user 'admin'     --inputs "{'username': 'server-login', 'ssh_key_data': '@~/.ssh/id_rsa'}" -k
{"inputs": {"ssh_key_data": ["Invalid certificate or key: @~/.ssh/id_rsa..."]}}
root@awxhost:/home/pix# cat ~/.ssh/id_rsa
-----BEGIN OPENSSH PRIVATE KEY-----
b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAABFwAAAAdzc2gtcn
NhAAAAAwEAAQAAAQEAs35Fwbp1VRbKAjw430a25KK0Bp/rWIdundzQJzarejIjGH3RXFVM

```

In my opinion I am not using a missing file...

Reply:
@mwiora when I test this on my local machine w/ my key, it _seems_ to create the credential w/ my key just fine?

```shell
[cli-ssh-example:] ~/dev/awx/awxkit awx credentials create --credential_type Machine --name 'My Fancy SSH Key' --user ryan --inputs '{"username": "server-login", "ssh_key_data": "@~/.ssh/id_rsa"}'
```

![image](https://user-images.githubusercontent.com/214912/66058424-5cbe8680-e508-11e9-91e4-aa6db60c6e4b.png)

Reply:
I guess you use an RH-based OS? I could have a try on RH on monday...

Reply:
It seems to work for me on both Fedora 28 and macOS Mojave.

Reply:
cloned current awx version as of today - (commit 52447f5) - and everything seems to be working now. As documentation has been cleared too, closing this PR.
Thanks